config/develop/nextflow-tower-config-bucket.yaml

```diff
@@ -0,0 +1,15 @@
+# {% set stack_name = "nextflow-tower-config-bucket" %}
```

Comment:
You don't! Cool huh?

Reply:
i guess that's because `#` isn't a comment in jinja however it looks like a comment because it's a .yaml file.  anyways might be nice to uncomment for readability but your call.

Reply:
@zaro0508 Sadly, in order for this to stay a yaml file I would have to disable yamllint for the whole file. I thought that using the comment trick was a convenient workaround. Instead I have converted this into a j2 file. I don't really like that solution either because now I'm not getting the benefit of linting either. What do you normally do so that you get linted but can add jinja statements?

Reply:
ahh, now that i know the intention behind the comment then i guess the comment is better than converting file to j2 to bypass the linter.  The other option you can do is to keep as j2 but also enable a jinja linter, here's how to do it with github actions https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/.github/workflows/aws-deploy.yaml#L19

Reply:
@zaro0508 In that case I'll go back to my comment. I would actually like having a jinja linter in place but it should go into pre-commit, not in the github action.